examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>Make sure that they actually line up. In the figures above there is a clear discrepancy at the middle site.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>